release/likely-commonjs.js

```diff
@@ -1343,6 +1346,7 @@ var Likely = function () {
                 this.timeout = setTimeout(this.ready.bind(this), this.options.timeout);
             } else {
                 this.appear();
+                this.ready();
```

Comment:
**Before**

`ready` is called only inside `updateCounter`, so when the counters are disabled and we don't call this function, we don't set `_ready` modifier and the tests fail (since they check it).

**After**

Now `ready` is called in `init` function. So it's ready all the time.

PS. It's still confusing to have both appear & ready but it doesn't make sense to touch it here.